### ISSUE TITLE
fullscreen: option to keep enabled addons (and/or sections) collapsed

### ISSUE THREAD

**Issue by ghost** [ScratchAddons/ScratchAddons#6961]
### Why this would be helpful

Having every enabled addon be uncollapsed can get quite cluttery, I know the search bar exists - However it's still really clutterly, especially with addons that have a lot of settings (prime examples would be `dark-www`, `editor-dark-mode`, `editor-theme3`)

### My suggestion

Have an option to collapse enabled addons or every section by default, or have them automatically collapsed when opening the fullscreen editor.
Enabled addons already start collapsed in the pop-up anyway. <sub>(I know the pop-up is really small, however it's still a lot less cluttery than having all of them uncollapsed)</sub>

### Possible alternatives

1. don't implement
2. Have all sections collapsed by default (though this would be weird UX-wise for a new user)
3. #1211

### Additional context

_No response._

**Comment by DNin01** [comment]
Another option is to separate the list of addons and their settings. Various other software with plugin managers such as Microsoft PowerToys do this. PowerToys, for example has a sidebar on the left with all of the modules listed as tabs which you can click to open their settings, in addition to a dashboard tab that just lists the modules' names, short descriptions, and on/off switches. I also shared a concept image for that kind of layout but for Scratch Addons in #6317.

**Comment by WorldLanguages** [comment]
> 2\. Have all sections collapsed by default (though this would be weird UX-wise for a new user)

Yeah, figuring out the best way to structure the settings page is quite a challenge. Users open it for different reasons and we don't know ahead of time if they are looking for a specific setting to change it, or if they're just exploring new addons, or something else. If the reader is interesting, you may use GitHub search to find some issues/discussions where we talk about this.
